app/src/components/profile/Profile.tsx

```diff
@@ -9,10 +9,10 @@ function Profile(props: Props) {
   //  더미 사용
   const profileImg = props.isLogin
     ? 'https://i.ibb.co/qjtcKYZ/profile.png'
-    : '/dummyUser.png';
+    : '/img/profile.png';
 
   const commonStyle = css`
-    background: no-repeat center/cover;
+    background: white no-repeat center/cover;
     background-image: url(${profileImg});

```

Comment:
```suggestion
    background: white url(${profileImg}) no-repeat center/cover;
```
background: 색상 이미지경로 반복 포지션/사이즈 의 값을 받는데 작성하지 않은 속성은 default 값으로 적용됩니다.